src/yb/common/wire_protocol.proto

```diff
@@ -117,6 +117,8 @@ message NodeInstancePB {
   // On a freshly initialized server, the first sequence number
   // should be 0.
   required int64 instance_seqno = 2;
+  // The time point when Tserver/master start up,it derives from Env::NowMicros()
```

Comment:
start up -> starts up

it derives -> it is derived

Reply:
No need .   i have tested on my local cluster .

Reply:
don't quite understand your reply.
I was pointing out grammatical mistake in comment.

Reply:
@tedyu  I have already fixed the  grammatical mistake in comment.

Reply:
Now I see the uo-to-date comment in code which is fine.

I don't have other review comment.